Formula/dbhash.rb

```diff
@@ -26,7 +26,14 @@ def install
     dbpath = testpath/"test.sqlite"
     sqlpath = testpath/"test.sql"
     sqlpath.write "create table test (name text);"
-    system "/usr/bin/sqlite3 #{dbpath} < #{sqlpath}"
+    sqlite = nil
+    on_macos do
+      sqlite = "/usr/bin/sqlite3"
+    end
+    on_linux do
+      sqlite = Formula["sqlite"].bin/"sqlite3"
+    end
+    system "#{sqlite} #{dbpath} < #{sqlpath}"
```

Comment:
This should work right? `/usr/bin` is in path, and `uses_from_macos` thould add it to path on linux
```suggestion
    system "sqlite #{dbpath} < #{sqlpath}"
```